### PR TITLE
Update RU translation

### PR DESCRIPTION
**Pull request by AAS**
Fix Book of the Blind verse and make more translations.

### CODE REVIEW

**Review by AJenbo — APPROVED** [review]

File: Translations/ru.po [hunk 1]
```diff
@@ -43,163 +43,165 @@ msgstr "Программисты"
...
 
 #: Source/DiabloUI/credits_lines.cpp:29
 msgid "Battle.net Programming"
-msgstr "Battle.net программисты"
+msgstr "Программисты Battle.net"
 
 #: Source/DiabloUI/credits_lines.cpp:32
 msgid "Serial Communications Programming"
-msgstr ""
+msgstr "Программирование последовательного порта"
```
Comment (Iniquitatis) [inline Translations/ru.po:54]: Не уверен, что кто-то программировал serial port. =)

File: Translations/ru.po [hunk 2]
```diff
@@ -43,163 +43,165 @@ msgstr "Программисты"
...
 
 #: Source/DiabloUI/credits_lines.cpp:35
 msgid "Installer Programming"
 msgstr "Программисты инсталятора"
```
Comment (Iniquitatis) [inline Translations/ru.po:58]: Можно было исправить на "инсталлятора".

File: Translations/ru.po [hunk 3]
```diff
@@ -43,163 +43,165 @@ msgstr "Программисты"
...
 
 #: Source/DiabloUI/credits_lines.cpp:52
 msgid "Cinematic Art Directors"
-msgstr ""
+msgstr "Художественные руководители роликов"
 
 #: Source/DiabloUI/credits_lines.cpp:55
 msgid "3D Cinematic Artwork"
-msgstr ""
+msgstr "3D художники роликов"
```
Comment (Iniquitatis) [inline Translations/ru.po:78]: Художники-то 3D, никто не сомневается.
"Художники 3D-роликов".

File: Translations/ru.po [hunk 4]
```diff
@@ -5973,46 +5975,46 @@ msgstr ""
...
 #: Source/stores.cpp:233
 msgid "Leave the shop"
-msgstr ""
+msgstr "Покинуть магазин"
 
 #: Source/stores.cpp:274 Source/stores.cpp:623 Source/stores.cpp:989
 #, c-format
 msgid "I have these items for sale:             Your gold: %i"
-msgstr ""
+msgstr "У меня есть на продажу:                Ваше золото: %i"
```
Comment (Iniquitatis) [inline Translations/ru.po:6017]: Один лишний пробел.

File: Translations/ru.po [hunk 5]
```diff
@@ -6021,70 +6023,71 @@ msgstr ""
...
 
 #: Source/stores.cpp:819
 msgid "You do not have enough gold"
-msgstr ""
+msgstr "У вас не достаточно золота"
 
 #: Source/stores.cpp:827
 msgid "You do not have enough room in inventory"
-msgstr ""
+msgstr "У вас не достаточно места в инвентаре"
```
Comment (Iniquitatis) [inline Translations/ru.po:6090]: "Недостаточно".

File: Translations/ru.po [hunk 6]
```diff
@@ -6200,47 +6204,47 @@ msgstr "Поговорить с %s"
...
 
 #: Source/stores.cpp:1137
 msgid "version"
-msgstr ""
+msgstr "версии"
 
 #: Source/stores.cpp:1164
 msgid "Gossip"
-msgstr ""
+msgstr "Сплети"
```
Comment (Iniquitatis) [inline Translations/ru.po:6223]: "Сплетни".

File: Translations/ru.po [hunk 7]
```diff
@@ -9897,23 +9900,23 @@ msgstr "Спуститься на уровень %i"
...
 #: Source/trigs.cpp:563
 #, c-format
 msgid "Up to Nest level %i"
-msgstr ""
+msgstr "Подняться в Гнездо на уровень %i"
 
 #: Source/trigs.cpp:681
 msgid "Down to Diablo"
-msgstr "Спустится в Диабло"
+msgstr "Спустится к Диабло"
```
Comment (Iniquitatis) [inline Translations/ru.po:9917]: "-ться".

File: Translations/ru.po [hunk 8]
```diff
@@ -9897,23 +9900,23 @@ msgstr "Спуститься на уровень %i"
...
 
 #: Source/trigs.cpp:718 Source/trigs.cpp:735 Source/trigs.cpp:752
 #, c-format
 msgid "Back to Level %i"
-msgstr "Вернутся на уровень %i"
+msgstr "Вернутся на Уровень %i"
```
Comment (Iniquitatis) [inline Translations/ru.po:9922]: "-ться".

**Comment by AJenbo** [comment]
@Iniquitatis I wasn't aware you could help with reviewing the RU translation, so I already merged it :(
Could you summit the suggestions as a new PR?

Would you be willing to review the future PRs to the Russian translation?

**Comment by Iniquitatis** [comment]
@AJenbo, unfortunately, I don't have much spare time lately, so I can't be sure if I review everything in time. Still, I could do some proofreading later.

**Comment by AJenbo** [comment]
Thanks for doing what you can :)

**Pull request merged**